### PR TITLE
Create faq-license.rst

### PR DESCRIPTION
**Pull request by justinegeffen**

### CODE REVIEW

File: source/about/faq-license.rst [hunk 1]
```diff
@@ -0,0 +1,312 @@
+License questions
```
Comment (jasonblais) [inline source/about/faq-license.rst:1]:
```suggestion
Open-source licensing questions
```

**Review by jasonblais — COMMENTED** [review]
Thank you Justine! I would recommend the following organization:

Three clear subsections

1. Open-source licensing, terms of use and privacy policy
 - only includes http://mattermost-docs-preview-pulls.s3-website-us-east-1.amazonaws.com/5824/about/faq-license.html#what-are-mattermost-s-policies-around-licensing-terms-of-use-and-privacy
2. Source-available licensing
  - includes all the questions from the previous source available license page
3. Open source derivative work of Mattermost
  - includes the questions from http://mattermost-docs-preview-pulls.s3-website-us-east-1.amazonaws.com/5824/about/faq-license.html#how-can-i-create-an-open-source-derivative-work-of-mattermost onward, up to and including `Does Mattermost answer questions about open source licenses authored by other organizations?`

I would also remove the questions around license keys, non-profit license and academic license, and keep them in the Enterprise Edition section.

File: source/about/faq-license.rst [hunk 2]
```diff
@@ -0,0 +1,308 @@
...
+
+EXCEPT AS OTHERWISE EXPRESSLY AGREED BY THE PARTIES IN WRITING, MATTERMOST MAKES NO WARRANTIES OR REPRESENTATIONS WITH RESPECT TO ANY MATTERMOST PRODUCTS, DOCUMENTATION OR SUPPORT, AND HEREBY DISCLAIMS ALL OTHER EXPRESS AND ALL IMPLIED WARRANTIES, INCLUDING BUT NOT LIMITED TO IMPLIED WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE, AND NON-INFRINGEMENT."
+
+Does Mattermost answer questions about open source licenses authored by other organizations?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+No, if you have questions about an open source license, please consult the original author, or FAQs they offer.
+
+
+Mattermost Service Agreements
```
Comment (jasonblais) [inline source/about/faq-license.rst:253]: Should this section onward be included as part of the "What are Mattermost's policies around licensing, terms of use, and privacy?" question?

**Review by jasonblais — CHANGES_REQUESTED** [review]
Thanks Justine! One comment in-line, plus the following:

1. Can you help include the question we added via https://github.com/mattermost/docs/pull/5863
2. Have we tested all redirects from old FAQs link correctly to the new FAQ links?

**Comment by justinegeffen** [comment]
Changes have been implemented, @jasonblais, sorry for the delay!

File: source/about/faq-license.rst [hunk 3]
```diff
@@ -0,0 +1,318 @@
...
+components are licensed under the original license provided by the owner of the
+applicable component."
+
+Open source derivative work of Mattermost
+-----------------------------------------
+
+How do I fork Mattermost?
+-------------------------
+
+If you wish to create a forked version of the Mattermost source code, you must comply with the applicable licenses under which the source code is made available. Historically, this was made clear via this link: https://docs.mattermost.com/about/faq-business.html#copyright-and-licensing-of-mattermost-server-open-source-code.
```
Comment (jasonblais) [inline source/about/faq-license.rst:243]: Just confirming the links in this FAQ work once the PR is merged?
 - https://docs.mattermost.com/about/faq-business.html#copyright-and-licensing-of-mattermost-server-open-source-code
 - https://docs.mattermost.com/about/faq-enterprise.html#what-are-mattermost-s-policies-around-licensing-terms-of-use-and-privacy

Not sure if they'll get redirected to the right section automatically

Comment (jasonblais) [reply source/about/faq-license.rst:243]: Thanks @justinegeffen! Just one question from my side

Comment (justinegeffen) [reply source/about/faq-license.rst:243]: I'll set those redirects up, thanks for the reminder.

File: source/conf.py [hunk 4]
```diff
@@ -78,6 +78,10 @@ def setup(_: Sphinx):
         "https://docs.mattermost.com/about/subscription.html#do-you-have-a-program-for-official-non-profits-open-source-projects-and-charities",
 "about/deployments-and-editions.html":
         "https://docs.mattermost.com/about/editions-and-offerings.html",
+"https://docs.mattermost.com/about/faq-business.html#copyright-and-licensing-of-mattermost-server-open-source-code":
```
Comment (jasonblais) [inline source/conf.py:81]: Sorry, one more 🙏 Would we also add the following redirect @justinegeffen ?

```
"https://docs.mattermost.com/about/faq-mattermost-source-available-license.html":
	"http://docs.mattermost.com/about/faq-license.html#source-available-licensing",
```

Comment (justinegeffen) [reply source/conf.py:81]: Thanks, Jason! Appreciate the catch (and formatting). :)

**Review by jasonblais — APPROVED** [review]

**Pull request merged**